agent_build_refactored/linux/scalyr-agent-2-config

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
```

Comment:
Direct copy of https://github.com/scalyr/scalyr-agent-2/blob/master/agent_build/linux/scalyr-agent-2-config